config/kernel-pde-data.m4

```diff
@@ -4,11 +4,11 @@ dnl # PDE is replaced by PDE_DATA
 dnl #
 AC_DEFUN([ZFS_AC_KERNEL_PDE_DATA], [
 	AC_MSG_CHECKING([whether PDE_DATA() is available])
-	ZFS_LINUX_TRY_COMPILE_SYMBOL([
+	ZFS_LINUX_TRY_COMPILE([
```

Comment:
It looks like `PDE_DATA()` first appeared in 3.10 and has always been an exported function, i.e. `EXPORT_SYMBOL(PDE_DATA).`

To be more specific, it was first added as an inline function in upstream commit 
d9dda78bad879595d8c4220a067fc029d6484a16
and then changed to non-inline exported function in upstream commit
c30480b92cf497aa3b463367a82f1c2fdc5c46e9
both of which first appeared in 3.10 in terms of release kernel, so `PDE_DATA()` has always been a exported function.